platform-workflow-module/src/main/java/com/softicar/platform/workflow/module/workflow/node/items/move/WorkflowNodeMoveItemsPopup.java

```diff
@@ -0,0 +1,127 @@
+package com.softicar.platform.workflow.module.workflow.node.items.move;
+
+import com.softicar.platform.common.container.comparator.OrderDirection;
+import com.softicar.platform.common.core.i18n.IDisplayString;
+import com.softicar.platform.core.module.CoreImages;
+import com.softicar.platform.dom.elements.DomRow;
+import com.softicar.platform.dom.elements.DomTable;
+import com.softicar.platform.dom.elements.button.DomButton;
+import com.softicar.platform.dom.elements.popup.DomPopup;
+import com.softicar.platform.dom.elements.select.value.DomEntitySelect;
+import com.softicar.platform.dom.event.DomEventType;
+import com.softicar.platform.dom.event.IDomEvent;
+import com.softicar.platform.dom.event.IDomEventHandler;
+import com.softicar.platform.dom.node.IDomNode;
+import com.softicar.platform.emf.EmfCssClasses;
+import com.softicar.platform.workflow.module.WorkflowI18n;
+import com.softicar.platform.workflow.module.workflow.AGWorkflow;
+import com.softicar.platform.workflow.module.workflow.item.move.WorkflowItemsMover;
+import com.softicar.platform.workflow.module.workflow.node.AGWorkflowNode;
+import com.softicar.platform.workflow.module.workflow.version.AGWorkflowVersion;
+import java.util.Collection;
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+class WorkflowNodeMoveItemsPopup extends DomPopup {
+
+	private final AGWorkflowNode sourceNode;
+	private final DomEntitySelect<AGWorkflowNode> nodeSelect;
+
+	public WorkflowNodeMoveItemsPopup(AGWorkflowNode sourceNode) {
+
+		this.sourceNode = sourceNode;
+		this.nodeSelect = new DomEntitySelect<>();
+		setCaption(WorkflowI18n.MOVE_WORKFLOW_ITEMS_TO_ANOTHER_WORKFLOW_NODE);
+		setSubCaption(WorkflowI18n.ARG1_ITEM_S_OF_SOURCE_WORKFLOW_NODE_ARG2.toDisplay(sourceNode.getAllWorkflowItems().size(), sourceNode.toDisplay()));
```

Comment:
The count could be outdated when the user clicks OK. Maybe workflow items should be loaded here and passed to the WorkflowItemsMover, so we ensure, that we only move as many items as we say here.
But some items could have been processed in the meantime as well...

Alternative:
If the user (of this action) can see how many items are currently in this node, we don't need to include the count in the subcaption.

Reply:
i dont see a big problem with having it in the subcaption, its used by admins only anyways